otter/test/convergence/test_service.py

```diff
@@ -933,9 +941,6 @@ def plan(dgs, servers, lb_nodes, now, build_timeout):
              noop)
         ]
 
-        # all the servers updated in cache in beginning
-        self.cache.append(thaw(deleted.json))
```

Comment:
Must make sure removing this line is okay because it represents a change in behavior for updating the cache.